### DISCUSSION TITLE
Error in OAuth authentication in production environment

### DISCUSSION THREAD

**Discussion by issey-jp** [supabase/supabase#9010] [Questions]
I am learning Supabase and am trying to sign in using Google OAuth authentication.
Authentication is successful locally `localhost:3000`.
However, on the site deployed to Netlify I get the following error.

`Uncaught (in promise) FetchError: 404  (/api/_supabase/session) at async Object.callback`

After executing signIn(), we were redirected to the Google account selection screen and confirmed redirection to the Netlify public site.

I have tried the following for this error

1. enter Netlify environment variables (`SUPABASE_KEY` and `SUPABASE_URL`)
2. enter the Netlify URL in Supabase > Authentication > Settings > User Sessions Site URL and Redirect URLs (`https://***.netlify.app`)

I have entered everything in both cases and get the same error. `redirectTo` is not used.

Ideally, both `localhost:3000` and `https://***.netlify.app` would be approved. However, from past discussions, I don't think that has been achieved yet.

**Comment by hf** [comment]
Hey,

`https://***.netlify.app` is not a valid wildcard pattern. You should be using `https://*.netlify.app/` (mind the last `/`). Let me know if you have further questions.

**Comment by issey-jp** [comment]
Dear Stojan

Thanks for your reply.
You are right, I tried the domain. (I am typing as you see in the image)
But the problem is not solved.

[image: image.png]

2022年9月21日(水) 17:44 Stojan Dimitrovski ***@***.***>:

**Reply by J0** [reply]
Hey @issey-jp 

Sorry to hear that the solution is not working for you -- do you mind sending over the image again so we can best troubleshoot? The image doesn't seem to render on my end unfortunately -- this is what it looks like for me

<img width="938" alt="CleanShot 2022-09-22 at 15 32 58@2x" src="https://user-images.githubusercontent.com/8011761/191747897-e20dc724-f639-4b2f-a612-401445e80f4a.png">

**Comment by issey-jp** [comment]
Dear.

Sorry , Uploaded the image again.Can you see this image?


[image: image.jpg]

2022年9月22日(木) 21:33 Joel Lee ***@***.***>:

**Reply by J0** [reply]
Hey @issey-jp,

Thanks for taking the time -- unfortunately the image still does not render for me -- this is what it looks like on my end:

<img width="945" alt="CleanShot 2022-09-24 at 20 05 47@2x" src="https://user-images.githubusercontent.com/8011761/192110230-6c920be8-ab6f-4b97-ac79-e5e0712da620.png">

**Comment by issey-jp** [comment]
Dear,

Uploaded to Google Drive. Can I see this one?

https://drive.google.com/drive/folders/1gfi20BxFtSsw7z_UfLWbLGVA-F8zy2fg?usp=sharing

2022年9月25日(日) 2:06 Joel Lee ***@***.***>:

**Comment by issey-jp** [comment]
This error message

[image: error.jpg]

2022年9月24日(土) 21:02 Issei Sakai ***@***.***>:
